### ISSUE TITLE
[production/AQM.v7] Chemical ICs generation with previous two or three cycles restart files

### ISSUE THREAD

**Issue by chan-hoo** [ufs-community/ufs-srweather-app#773]
Copied from: https://github.com/NOAA-EMC/AQM/issues/83

The NRT run failed at 06z cycle today because of 00z-cycle failure and no restart files to be used by the 06z cycle. In this case, please add a new feature of using the previous day's 12z cycle restart files to generate chemical IC files for today 06z cycle since yesterday's 18z cycle run had only 6-hr restart file.

We need to do the same thing for day 12z cycle run if today 06z cycle restart files are missing. In this case, we have to previous day 12z cycle restart files to generate chemical ICs for today 12z cycle.

Please be reminded that 00z and 18z cycles have 6-hr forecasts only.

Thanks,

Jianping

**Comment by chan-hoo** [comment]
@JianpingHuang-NOAA, for this capability, we'll need the restart files at fhr=18, so the configuration file `config.yaml` should be modified as follows:
```
  RESTART_INTERVAL: 6 18 24 42 60
```

**Comment by JianpingHuang-NOAA** [comment]
That is right. Thanks

**Comment by chan-hoo** [comment]
@JianpingHuang-NOAA, your description of failure doesn't make sense to me. Currently, the task 'aqm_ics' has the dependency as follows:
```
          <datadep age="00:00:00:05"><cyclestr offset="-{{- cycl_freq -}}">&COMIN_DIR;</cyclestr>/RESTART/<cyclestr>@Y@m@d.@H@M@S.fv_tracer.res.tile1.nc</cyclestr></datadep>
```
This means that the task `aqm_ics` will be triggered only when the above restart file exists. However, you mentioned that your run failed due to the missing restart file. Your run should not fail but be on hold. I guess your failure might be caused by another reason. Moreover, if the '00z' cycle fails, your run will be stopped with the `failure` message in the current rocoto system. I don't understand how the `06z` cycle keeps running with the failure of `00z`.

**Comment by chan-hoo** [comment]
@JianpingHuang-NOAA, if you don't have any other comments, I'll close this issue.

**Comment by JianpingHuang-NOAA** [comment]
@Chan-Hoo Jeon - NOAA Affiliate ***@***.***>  I did not get
time to test it recently. I will let you know once the test is done. Thanks
!

**Comment by chan-hoo** [comment]
@JianpingHuang-NOAA, I didn't update anything about this. My comment was that this capability would not be available as far as I know. If you have any idea for this, I'll change the assignee to you and wait for your PR.
